src/docs/asciidoc/reference/configuration/configuration-properties.adoc

```diff
@@ -1,6 +1,26 @@
 :relative-path: ../../
 include::{docdir}/variables.adoc[]
 
+ZetaPush provides the `application.json` file to let you configure the behavior of your application.
 
+For now, the `application.json` file is mainly used for the <<reference#integrate_user_management, Standard User Workflow>> but you can also <<reference#configuration_properties ,configure your own properties for your application>>.
+The list of available properties for the <<reference#integrate_user_management, Standard User Workflow>> are listed in <<reference#customize_standard_user_workflow, this section>>.
+
+.New properties will added
+[NOTE]
+====
+New _Cloud Services_ will be developed (to handle data, security, etc...).
+With them, some available properties of the `application.json` will be added.
+====
+
+.Futur features
+[NOTE]
+====
+New features about `application.json` file will be soon released :
+
+* Support of multi `application.json` file, one per worker (when the multi-worker) will be released.
+* Support of `application.json` file per environment like `application-<env>.json` for example. (When the multi-environment will be released)
+* Autocompletion of properties in VSCode (and other IDE) 
+====
```

Comment:
We can also add a [TODO] to indicate that developer can provide its own configuration property source (for example, download properties from an external service or from a git repository and configure priority of this source)